dhcpv4/dhcpv4.go

```diff
@@ -574,7 +580,8 @@ func (d *DHCPv4) RootPath() string {
 //
 // The Bootfile Name option is described by RFC 2132, Section 9.5.
 func (d *DHCPv4) BootFileNameOption() string {
-	return GetString(OptionBootfileName, d.Options)
+	name := GetString(OptionBootfileName, d.Options)
+	return strings.TrimRight(name, "\x00")
```

Comment:
Do the same for the TFTPServerName option?

Reply:
done